tests/adapters_t/batchsystems_t/test_htcondor.py

```diff
@@ -32,26 +32,26 @@ def setUp(self):
         self.cpu_ratio = 0.9
         self.memory_ratio = 0.8
         self.command = (
-            "condor_status -af:t Machine State Activity TardisDroneUuid "
+            "condor_status -af:t Machine Name State Activity TardisDroneUuid "
```

Comment:
I don't know the commands, but is this correct here that `Machine` as well as `Name` are given as input? Same below.

Reply:
This are just the column names we would like to get from the `condor_status` command. We need the `Machine` column for backward compatibility in https://github.com/MatterMiners/tardis/blob/546a3b5ca10654766a6a66dcab2f98cc86c5d9c9/tardis/adapters/batchsystems/htcondor.py#L54 as well. Issue to change this is already created #101.